src/qtgui/dockfft.ui

```diff
@@ -6,242 +6,320 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>368</width>
-    <height>444</height>
+    <width>250</width>
+    <height>250</height>
    </rect>
   </property>
   <property name="sizePolicy">
-   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
+   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
-    <width>327</width>
-    <height>200</height>
+    <width>250</width>
+    <height>250</height>
    </size>
   </property>
-  <property name="maximumSize">
-   <size>
-    <width>524287</width>
-    <height>524287</height>
-   </size>
-  </property>
-  <property name="windowIcon">
-   <iconset resource="../../resources/icons.qrc">
-    <normaloff>:/icons/icons/signal.svg</normaloff>:/icons/icons/signal.svg</iconset>
-  </property>
-  <property name="toolTip">
-   <string>FFT performance and display settings</string>
-  </property>
-  <property name="allowedAreas">
-   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
-  </property>
   <property name="windowTitle">
    <string>FFT Settings</string>
   </property>
   <widget class="QWidget" name="dockWidgetContents">
+   <property name="sizePolicy">
+    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
+     <horstretch>0</horstretch>
+     <verstretch>0</verstretch>
+    </sizepolicy>
+   </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
-     <widget class="QScrollArea" name="scrollArea">
```

Comment:
Removing the `QScrollArea` from the FFT Settings panel increases the minimum height of the Gqrx window. Since FFT Settings is the tallest panel, it might be good to leave the `QScrollArea` in.

Reply:
Agree.

Reply:
Prior to this change, the minimum height was a bit under 720 pixels. I think that's a good number to shoot for, to keep Gqrx usable on small screens.